targets/arduino_101/src/libc_support.c

```diff
@@ -0,0 +1,105 @@
+/* Copyright 2016 Samsung Electronics Co., Ltd.
```

Comment:
I still don't fully understand why this file (`libc_support.c`) is included. In the `prj.conf` you are using the newlib libc library which it not the Zephyr minimal nor JerryScript libc. The newlib comes from the compilers that Zephyr supports.